### PR TITLE
Remove dead code

### PR DESCRIPTION
**Pull request by kamil-tekiela**

### CODE REVIEW

File: libraries/classes/Core.php [hunk 1]
```diff
@@ -125,24 +125,6 @@ public static function warnMissingExtension(
...
     }
 
-    /**
-     * returns count of tables in given db
-     *
-     * @param string $db database to count tables for
-     *
-     * @return int count of tables in $db
-     */
-    public static function getTableCount(string $db): int
```
Comment (williamdes) [inline libraries/classes/Core.php:135]: in 4.6.6 a test tested it
The last version to use it was 3.5.8: https://github.com/phpmyadmin/phpmyadmin/blob/RELEASE_3_5_8/libraries/List_Database.class.php#L291

File: libraries/classes/Display/Results.php [hunk 2]
```diff
@@ -93,10 +93,6 @@ class Results
     public const DISPLAY_FULL_TEXT = 'F';
     public const DISPLAY_PARTIAL_TEXT = 'P';
 
-    public const HEADER_FLIP_TYPE_AUTO = 'auto';
-    public const HEADER_FLIP_TYPE_CSS = 'css';
-    public const HEADER_FLIP_TYPE_FAKE = 'fake';
-
```
Comment (williamdes) [inline libraries/classes/Display/Results.php:99]: one of the last versions to use then was 4.2.12

https://github.com/phpmyadmin/phpmyadmin/blob/RELEASE_4_2_12/libraries/DisplayResults.class.php#L2204-L2210

File: libraries/classes/Display/Results.php [hunk 3]
```diff
@@ -112,9 +108,6 @@ class Results
     public const ALL_ROWS = 'all';
     public const QUERY_TYPE_SELECT = 'SELECT';
 
-    public const ROUTINE_PROCEDURE = 'procedure';
-    public const ROUTINE_FUNCTION = 'function';
-
```
Comment (williamdes) [inline libraries/classes/Display/Results.php:117]: same
https://github.com/phpmyadmin/phpmyadmin/blob/RELEASE_4_2_12/libraries/DisplayResults.class.php#L3230-L3231

File: libraries/classes/Html/Generator.php [hunk 4]
```diff
@@ -151,35 +151,6 @@ public static function getDbLink(string $database): string
...
-    /**
-     * Prepare a lightbulb hint explaining a known external bug
-     * that affects a functionality
-     *
-     * @param string $functionality  localized message explaining the func.
-     * @param string $component      'mysql' (eventually, 'php')
-     * @param string $minimumVersion of this component
-     * @param string $bugReference   bug reference for this component
-     */
-    public static function getExternalBug(
```
Comment (williamdes) [inline libraries/classes/Html/Generator.php:163]: The last version to use it was 4.7.9:
https://github.com/phpmyadmin/phpmyadmin/blob/RELEASE_4_7_9/libraries/display_export.lib.php#L398

File: libraries/classes/ConfigStorage/Relation.php [hunk 5]
```diff
@@ -560,31 +560,6 @@ public function getDbComment(string $db): string
         return '';
     }
 
-    /**
-     * Gets the comment for a db
-     *
-     * @return mixed[] comments
-     */
-    public function getDbComments(): array
```
Comment (williamdes) [inline libraries/classes/ConfigStorage/Relation.php:568]: had a test in 4.6.6
The last version to use it was 3.5.8:
https://github.com/phpmyadmin/phpmyadmin/blob/RELEASE_3_5_8/libraries/List_Database.class.php#L233

**Review by williamdes — APPROVED** [review]
Thank you !
I added historical references to ensure we are not deleting a functionality by mistake

**Comment by kamil-tekiela** [comment]
Thank you for doing the research. I appreciate the comments.

**Review by MauricioFauth — APPROVED** [review]

**Pull request merged**